src/Sylius/Bundle/ShopBundle/Resources/views/Order/_summary.html.twig

```diff
@@ -13,7 +13,7 @@
                     {{ money.convertAndFormat(order.total) }}
                 </div>
                 <div class="item">
-                    {{ order.items|length }} {{ 'sylius.ui.items'|trans|lower }}
+                    {{ order.totalQuantity }} {{ 'sylius.ui.items'|trans|lower }}
```

Comment:
I'm wondering, maybe we should change the `'sylius.ui.items'` translation as well? I agree that there we should have a total quantity of the order, but then it should be something like `4 units` or `4 products` or whatever, instead of `4 items` as order item can be indeed 1 :)

Reply:
I think `items` is the good word here.
It is the same word used in Amazon cart page even if you order several time the same product.
![amazon com shopping cart](https://user-images.githubusercontent.com/473193/48311430-269ab000-e5a0-11e8-9a18-19f3717bc632.png)

@Zales0123 let me know if it is ok for you